stetho/src/main/java/com/facebook/stetho/inspector/helper/ObjectIdMapper.java

```diff
@@ -0,0 +1,79 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.helper;
+
+import android.util.SparseArray;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.annotation.concurrent.GuardedBy;
+
+public class ObjectIdMapper {
```

Comment:
Perhaps you're right.  Even the Console tab use case will treat it as `Object`.  Nevermind :)